ui.frontend/src/view/FormFieldBase.js

```diff
@@ -437,6 +437,7 @@ class FormFieldBase extends FormField {
         if (this.errorDiv) {
             this.toggle(valid, Constants.ARIA_INVALID, true);
             this.element.setAttribute(Constants.DATA_ATTRIBUTE_VALID, valid);
+            this.widget.setAttribute(Constants.ARIA_INVALID, !valid);
```

Comment:
We are already doing a toogle on line 438, why is this copy/paste. If this is specific to component, then add it in their respective view code

Reply:
@rismehta line 438 adds aria-invalid attribute to the parent div. Its usage on non-form elements like <div> may not be semantically correct according to accessibility best practices. Moreover, the requirement is to add aria-invalid to all the widgets(input field). This is the reason I have added line 440. We can remove line 438 as it only adds aria invalid to the parent div, for this we have data-cmp-valid attribute as well on the parent div